BackofficeBundle/Resources/public/ecmascript/OpenOrchestra/Application/View/OrchestraView.js

```diff
@@ -12,8 +13,18 @@ class OrchestraView extends Backbone.View
      * @param {object}   parameters
      */
     _renderTemplate(templateName, parameters = {}) {
+        parameters = _.extend({renderTemplate: this._renderTemplate}, parameters);
```

Comment:
parameters = _.extend(true, {}, {renderTemplate: this._renderTemplate}, parameters);
first parameter true, to allow deep merge, second parameter {} to avoid changing parameter out of this scope.